src/net_processing.cpp

```diff
@@ -3838,7 +3838,7 @@ class CompareInvMempoolOrder
 bool PeerLogicValidation::SendMessages(CNode* pto)
 {
     const Consensus::Params& consensusParams = Params().GetConsensus();
-    {
+
```

Comment:
For future work, do we still need `cs_sendProcessing` ? It only guards `m_next_addr_send/m_next_local_addr_send`, both of them only RW in `SendMessages`, inside `ThreadMessageHandler`. 

Is this a relic from the past or do you see another reason ?

Reply:
I've removed it from `m_next_addr_send` and `m_next_local_addr_send` in https://github.com/jnewbery/bitcoin/commits/2020-06-cs-main-split. It could theoretically also be removed from guarding `SendMessages()` since we only ever call that from the message handler thread.